src/codegen/codegen_c_visitor.cpp

```diff
@@ -1412,13 +1412,25 @@ void CodegenCVisitor::print_function_call(const FunctionCall& node) {
     auto arguments = node.get_arguments();
     printer->add_text("{}("_format(function_name));
 
+    bool possible_nt_variable_types;
```

Comment:
```suggestion
    bool possible_nt_variable_types{false};
```
Maybe better to avoid the issue reported by the GitHub Actions bot?